2-structured-data/package.json

```diff
@@ -40,7 +40,6 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
-    "gcloud": "~0.37.0",
     "@google-cloud/datastore": "^0.1.1",
```

Comment:
I wasn't planning to - but if I can just drop it in and go (without any additional tests, since per SemVer it should be backwards compatible), I happily would. :+1: